saleor/checkout/utils.py

```diff
@@ -703,6 +703,38 @@ def fetch_checkout_lines(checkout: Checkout) -> Iterable[CheckoutLineInfo]:
     return lines_info
 
 
+def fetch_checkout_info(
```

Comment:
Let's try to move the fetching logic as well as the data class definition to a separate file for example `saleor/checkout/fetch.py`.

Reply:
I'll move it, in the separate PR.

Reply:
Sounds good :)